lib/events.js

```diff
@@ -166,7 +162,8 @@ EventEmitter.setMaxListeners =
         isEventTarget = require('internal/event_target').isEventTarget;
 
       // Performance for forEach is now comparable with regular for-loop
```

Comment:
Note to self: this comment should be updated or removed.

Reply:
removed I would say, they are not comparable at all.